spm_funcs.py

```diff
@@ -54,8 +54,26 @@ def get_spm_globals(fname):
     spm_vals : array
         SPM global metric for each 3D volume in the 4D image.
     """
-    # +++your code here+++
-    # return
+    # Load the 4D image using nibabel
+    img = nib.load(fname)
+
+    # Initialize an empty list to store SPM metric values
```

Comment:
Maybe too many comments?  Too many comments can make the code harder to read, because there is more text.   Still, too many comments is almost invariable better than not enough.